Source/CsDebugScript.Engine/Engine/Utility/SimpleCache.cs

```diff
@@ -91,6 +94,14 @@ public T Value
                 Cached = true;
             }
         }
+
+        /// <summary>
+        /// Invalidate cache entry.
+        /// </summary>
+        public void InvalidateCache()
+        {
+            Cached = false;
+        }
```

Comment:
I kept the current solution for now. I also added more thorough clearing of caches in process task (reflection that takes all simple caches and calls this method).

---
In reply to: [69630848](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69630848) [](ancestors = 69630848)